R/dockerfile.R

```diff
@@ -76,7 +76,7 @@
 
 .write_dockerfile <- function(dockerfile_content, path) {
```

Comment:
Actually, I would argue that all directives (FROM, ENV, COPY, RUN, COPY, CMD) should be wrapped.

Therefore, it is possible to

```r
unlist(lapply(dockerfile_content, .run_line_wrap))
```

But again, we need tests to see how it works.